backends/ebpf/ebpfTable.h

```diff
@@ -38,44 +58,84 @@ class EBPFTableBase : public EBPFObject {
                   CodeGenInspector* codeGen) :
             program(program), instanceName(instanceName), codeGen(codeGen) {
         CHECK_NULL(codeGen); CHECK_NULL(program);
-        keyTypeName = program->refMap->newName(instanceName + "_key");
-        valueTypeName = program->refMap->newName(instanceName + "_value");
+        keyTypeName = instanceName + "_key";
```

Comment:
The original code made sure that no such variables exist already in the code.
What would happen if they do exist?

Reply:
I think `newName()` is useful for objects' instances, because it protects from conflicting names if we have multiple lookups to a table. I don't see a reason why we would need a unique key and value **type** - since key and value types are specific to a P4 table. Does it make sense?

Reply:
For any P4 program the user writes the generated C should be unambiguous. There should be no way the user program names a table (or some other construct) in such a way that your generated names collide.